test/Microsoft.AspNet.Mvc.ViewFeatures.Test/Rendering/HtmlHelperFormTest.cs

```diff
@@ -477,7 +516,46 @@ public void BeginForm_EndForm_SuppressAntiforgeryToken()
             Assert.NotNull(writer);
 
             // Act & Assert
-            using (var form = htmlHelper.BeginForm(FormMethod.Post, suppressAntiforgery: true, htmlAttributes: null))
+            using (var form = htmlHelper.BeginForm(FormMethod.Post, antiforgery: false, htmlAttributes: null))
```

Comment:
Add a test demonstrating that `antiforgery: true` "wins" even if `method: FormMethod.Get`. That is the `<form> element will contain an `<antiforgery>` element in that case.